src/pocketmine/Player.php

```diff
@@ -3241,7 +3241,14 @@ public function handleItemFrameDropItem(ItemFrameDropItemPacket $packet) : bool{
 
 		$tile = $this->level->getTile($this->temporalVector->setComponents($packet->x, $packet->y, $packet->z));
 		if($tile instanceof ItemFrame){
+			$ev = new PlayerInteractEvent($this, $this->inventory->getItemInHand(), $tile->getBlock(), $tile->getBlock()->getDamage(), PlayerInteractEvent::LEFT_CLICK_BLOCK);
```

Comment:
Would
```
$face = 5 - $damage;
```
be a better fix or should I just copy $faces array and then search the array?